static/app/views/performance/newTraceDetails/traceModels/traceTree.tsx

```diff
@@ -562,7 +568,13 @@ export class TraceTree {
               parent.parent_transaction?.metadata.project_slug,
       });
 
-      node.canFetch = true;
+      // We check for at least 2 spans because the first span is the transaction itself.
+      if (isTransactionNode(node)) {
+        const spanChildrenCount =
+          metaResults?.data?.transactiontoSpanChildrenCount[node.value.event_id];
+        node.canFetch = spanChildrenCount ? spanChildrenCount >= 2 : false;
+      }
```

Comment:
If the result of metaResults.data lookup is undefined, then node.canFetch has to return true. It can only return false if the key is present in the lookup and the value is >= 2